File: tests/fixtures/php-basic/Sample.php
<?php

class Sample
{
    public function square(int $n): int
    {
        return $n * $n;
    }
}

interface Greeter
{
    public function greet(): string;
}

trait Loggable
{
    public function log(): void {}
}

enum Suit
{
    case Hearts;
}

function helper(): int
{
    return 42;
}

File: tests/fixtures/java-calls/Sample.java
package com.app;

class Sample {
  int square(int x) {
    return mul(x, x);
  }

  int mul(int a, int b) {
    return a * b;
  }

  int recurse(int n) {
    return n <= 1 ? 1 : recurse(n - 1);
  }

  void log() {
    System.out.println(square(2));
  }
}
